roles/download/defaults/main.yml

```diff
@@ -1371,6 +1384,21 @@ downloads:
     groups:
     - k8s_cluster
 
+  ciliumcli:
+    enabled: "{{ kube_network_plugin == 'cilium' or cilium_deploy_additionally | default(false) | bool }}"
+    file: true
+    version: "{{ cilium_cli_version }}"
+    dest: "{{ local_release_dir }}/cilium"
+    sha256: "{{ ciliumcli_binary_checksum }}"
+    url: "{{ ciliumcli_download_url }}"
+    mirrors:
```

Comment:
The cilium cli may not needs mirror.
Only one mirror URL is not reasonable.

Reply:
> The cilium cli may not needs mirror. Only one mirror URL is not reasonable.

THX